src/libtiled/isometricrenderer.cpp

```diff
@@ -401,6 +401,7 @@ void IsometricRenderer::drawMapObject(QPainter *painter,
             painter->setPen(pen);
             painter->drawRect(rect);
             pen.setStyle(Qt::DotLine);
+            // since now layers will have a tint color, should we remove the object layer color property for objects and use the layer tintcolor here? or since object layers already have a color property, shoul we stick with the name Color and rename tintColor to color? aniways the layer color would have to be provided by the caller of IsometricRenderer::drawMapObject
```

Comment:
It makes sense to keep tint color separate from this color since they do entirely different things.

I believe the layer tint color can be obtained through the object using `object->objectGroup()->tintColor()`, as per previous suggested change.

Reply:
done